packages/shared/src/lib/abstractBoard/boardFactory.ts

```diff
@@ -49,6 +52,11 @@ export interface SierpinskyBoardConfig {
   size: number;
 }
 
+export interface GridWithHolesBoardConfig {
+  type: "gridWithHoles";
+  bitmap: (0 | 1 | 2 | 3 | 4)[][];
```

Comment:
I'm not sure I understand the bitmap. What do the different values represent?

Reply:
Might be explained by this forum post (and the two after): https://forums.online-go.com/t/collective-development-of-a-server-for-variants/43682/145

Reply:
I've added a comment to explain the values.